tests/resources/functional/teradata/ddl/test_map.sql

```diff
@@ -0,0 +1,46 @@
+-- teradata sql:
+CREATE MULTISET TABLE NEW_TABLE_NAME ,FALLBACK ,
+     NO BEFORE JOURNAL,
+     NO AFTER JOURNAL,
+     CHECKSUM = DEFAULT,
+     DEFAULT MERGEBLOCKRATIO,
+     MAP = TD_MAP1
```

Comment:
@sriram251-code in the code, add links to documentation which defines what MAP and Compress does that way it is easy to navigate to understanding why we have ignored them.